content-repo/extra-docs/packs/Email-Communication-Readme.md

```diff
@@ -70,58 +70,6 @@ The Email Communication content pack includes several content items.
 * **Email Communication**: Used in the **Email Communication** incident type.
 * **Email Threads**: This layout can be added to other incident types to add email functionality to them.

```

Comment:
Maybe let's add a link to their section (for example for the email communication - add hyperlink to the Layout Instructions  -> Email Communication Layout